### PR TITLE
Use large border-radius in Action Button

### PR DESCRIPTION
**Pull request by GretaD**
ref: https://github.com/nextcloud/mail/issues/6200#issuecomment-1108416443

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
:+1: :elephant:

**Comment by ChristophWurst** [comment]
Ready for review or not?

**Comment by raimund-schluessler** [comment]
@GretaD I had a bit of time and worked on your PR. I hope, you don't mind. If you don't like my changes, just discard them.

The result now looks like this:
Screenshot 2022-05-30 at 23-43-31 Tasks - Nextcloud

The values of the border radius might need some tweaking, though. Given that we should respect OuterRadius = InnerRadius + Padding, the OuterRadius is 14 px at the moment, which might be a bit large. If we want it smaller, we would need to reduce either padding and/or InnerRadius.

@nimishavijay @jancborchardt

File: src/components/Actions/Actions.vue [hunk 1]
```diff
@@ -170,6 +170,7 @@ export default {
 			:placement="placement"
 			:boundaries-element="boundariesElement"
 			:container="container"
+			popover-base-class="action-item__popover"
```
Comment (raimund-schluessler) [inline src/components/Actions/Actions.vue:173]: Necessary, so we can only style popovers from `Actions`.

File: src/components/Popover/Popover.vue [hunk 2]
```diff
@@ -77,6 +77,13 @@ export default {
 		VPopover,
 	},
 
+	props: {
+		popoverBaseClass: {
```
Comment (raimund-schluessler) [inline src/components/Popover/Popover.vue:81]: Needs some documentation.

**Comment by raimund-schluessler** [comment]
> The values of the border radius might need some tweaking, though. Given that we should respect OuterRadius = InnerRadius + Padding, the OuterRadius is 14 px at the moment, which might be a bit large. If we want it smaller, we would need to reduce either padding and/or InnerRadius.

I adjusted the border radii. This looks like this now:
Screenshot 2022-05-31 at 07-26-35 Tasks - Nextcloud

This is closer to what @nimishavijay had in mind in https://github.com/nextcloud/mail/issues/6200#issuecomment-1108416443, I guess.

**Comment by GretaD** [comment]
> Ready for review or not?

No, sorry, my first approach was not the perfect one and raimund had a good point on fixing everything here on vue instead of getting sever involved

**Comment by GretaD** [comment]
> @GretaD I had a bit of time and worked on your PR. I hope, you don't mind. If you don't like my changes, just discard them.
> 
No problem, i was working on this yesterday and i did smth similar but the row jumped a bit on hover, thats why i didnt push. 
Is this ready for review? I checked the code and looks good but didnt test with mail or calendar

**Comment by GretaD** [comment]
works good with mail too

**Review by st3iny — APPROVED** [review]
I tested it on calendar and it worked well. It looks very good too.

~I couldn't test on mail because the app won't load and I get the following error in the console:~
```
TypeError: __webpack_modules__[moduleId] is not a function
    __webpack_require__ isMobile.js:107
```
~But I don't think that's related.~

**EDIT:** Nevermind, I did `npm ci` and `npm run dev` again and it worked.

**Comment by ChristophWurst** [comment]
Shall we tag this as breaking? I don't know how high impact design changes are handled here

**Pull request merged**